.ci.yaml

```diff
@@ -90,6 +90,18 @@ platform_properties:
       device_type: none
       cpu: arm64
       xcode: 14a5294e # xcode 14.0 beta 5
+  mac_benchmark:
+    properties:
+      dependencies: >-
+        [
+          {"dependency": "apple_signing", "version": "version:2022_to_2023"}
+        ]
+      device_type: none
+      mac_model: "Macmini8,1"
```

Comment:
Are these meant for x86 benchmarks? The M1's I believe are "Macmini9,1"

Reply:
Yes, these benchmarks need to run on x64 Intel machines.

Reply:
These are desktop benchmarks and should always run on the same hardware to get consistent performance.  When I swapped all the hostonly bots to ARM I had to revert it because the macOS benchmarks started running on multiple models.  This change pegs it to the devices they were already running on before the ARM Macs were added.